xproc/src/main/xml/specification.xml

```diff
@@ -6545,6 +6545,10 @@ draft:</para>
         <para>Updated the description of the <tag class="attribute">select</tag> attribute for <tag>p:input</tag>
         so that it is in line with the more recent changes that have been applied to <tag>p:with-input</tag>.</para>
       </listitem>
+      <listitem>
+        <para>Changed description of <tag>p:viewport</tag> stating that the base URI of every matched note is the
+        document's base URI, not just for document and element nodes.</para>
```

Comment:
note → node

Reply:
Thanks! "note" is from the other project I am working on. :-)